composer.json

```diff
@@ -37,6 +37,9 @@
         "symfony/dotenv": "^6.4 || ^7.1",
         "symfony/var-dumper": "^6.4 || ^7.1"
     },
+    "conflict": {
+        "mongodb/mongodb": "<1.20"
+    },
```

Comment:
this is not needed if the version constraint is hire anyways - composer would never install something below 1.20, right?

Reply:
If you install it without require dev, nothing prevents you from installing 1.19 IMHO

Reply:
True thing, meaning as consequences we would need that for all suggestions?

And on the other hand, the dependency is optional - so a hard conflict i also a lie. Imagine using the lib without mongodb store but having mongodb installed for other parts of the app ...

Reply:
Nvm, getting into too much of theory